packages/block-editor/src/components/inserter/menu.native.js

```diff
@@ -122,10 +145,12 @@ function InserterMenu( {
 	 */
 	function getItems() {
 		// Filter out reusable blocks (they will be added in another tab)
-		const itemsToDisplay = items.filter(
+		let itemsToDisplay = items.filter(
 			( { name } ) => name !== 'core/block'
 		);
 
+		itemsToDisplay = searchItems( itemsToDisplay, filterValue );
```

Comment:
Minor thing, but I wonder if instead of using a let here we could used two `const` variables with names that showed how they were different.